### ISSUE TITLE
R - library() vs require()

### ISSUE THREAD

**Issue by chendaniely** [swcarpentry/bc#664]
under 06-best practices it mentions using both.

<code>library()</code> and <code>require()</code> are a little different, not sure if you want to mention the differences to novices when writing code

Yihui (creator of knitr, and on the rstudio team) is strongly against using <code>require()</code>
https://twitter.com/xieyihui/status/484476578416455680
one of the twitter replies he makes:

```
require() == inherits(try(library()), 'try-error')
```

his blog post about it: http://yihui.name/en/2014/07/library-vs-require/

the tl;dr: <code>require()</code> returns a True/False value and is used for within functions
